Taxopark/Taxopark/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Taxopark.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
```

Comment:
I am not sure that here is necessary to point the route. I think 'controller' route is set by default

Reply:
By default we have attribute-based controller routing - in Startup.cs, in ConfigureServices, the endpoints.MapControllers() method is invoked. This method allows you to access the controller by request _"localhost:хххх//weatherforecast"_, because the routing attribute _[Route("[controller]")]_ is applied to the controller. If we remove this attribute, we will get an exception.
![routerxception](https://user-images.githubusercontent.com/60783800/150340372-2c80f93b-84d4-47eb-b514-c21aebf85bd0.JPG)

Reply:
Thank you for explanation